test/fuzzer/fuzzer.cpp

```diff
@@ -230,7 +232,7 @@ fuzz_input populate_input(const uint8_t* data, size_t data_size) noexcept
     if (data_size < input_size_16bits)  // Not enough data for input.
         return in;
 
-    in.rev = rev_4bits > EVMC_PETERSBURG ? EVMC_PETERSBURG : static_cast<evmc_revision>(rev_4bits);
+    in.rev = rev_4bits > latest_rev ? latest_rev : static_cast<evmc_revision>(rev_4bits);
```

Comment:
Why not fuzz for older versions though?

Reply:
It does when `rev_4bits < latest_rev`. Only the `latest_rev` happens more frequently.